### PR TITLE
ref(replays): remove StringWalker from index and details

### LINKED ISSUE #56868: Remove URLWalker
### Problem
Kind of sucks in all regards

### Acceptance Criteria
- Remove it from the index table
- Remove it from the details header

### PR DESCRIPTION
**Pull request by michellewzhang**
Closes  https://github.com/getsentry/sentry/issues/56868

SCR-20231002-kqkn
SCR-20231002-kqmm

### CODE REVIEW

File: static/app/views/replays/detail/page.tsx [hunk 1]
```diff
@@ -71,15 +60,7 @@ function Page({
...
-          displayEmail={
-            <Cols>
-              {frames?.length ? (
-                <FrameWalker replayRecord={replayRecord} frames={frames} />
-              ) : (
-                <StringWalker urls={replayRecord.urls} />
-              )}
-            </Cols>
-          }
+          displayEmail={<Cols>{undefined}</Cols>}
```
Comment (ryan953) [inline static/app/views/replays/detail/page.tsx:63]: i'm guessing this doesn't break 🤞 

```suggestion
          displayEmail={undefined}
```

File: static/app/views/replays/replayTable/tableCell.tsx [hunk 2]
```diff
@@ -415,7 +411,6 @@ const StyledIconDelete = styled(IconDelete)`
   margin: ${space(0.25)};
 `;
 
-// Need to be full width for StringWalker to take up full width and truncate properly
 const UserBadgeFullWidth = styled(UserBadge)`
   width: 100%;
 `;
```
Comment (ryan953) [inline static/app/views/replays/replayTable/tableCell.tsx:416]: Can we remove UserBadgeFullWidth and use `<UserBadge>` directly now?

**Review by ryan953 — APPROVED** [review]
woo! all that glorious walker code is gone!

i'll miss it, but not really.

File: static/app/views/replays/detail/page.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/views/replays/detail/page.tsx:63]: so removing `Cols` displays the email, it's weird
SCR-20231003-mwsv

but when `Cols` is there, no email

Comment (ryan953) [reply static/app/views/replays/detail/page.tsx:63]: ahh. thanks for checking.

can we put in a `<div/>` instead, then we don't need the extra `styled` call

File: static/app/views/replays/replayTable/tableCell.tsx (hunk 2 above)
Comment (michellewzhang) [reply static/app/views/replays/replayTable/tableCell.tsx:416]: i think so!

**Pull request merged**